packages/core/src/controllers/ApiController.ts

```diff
@@ -230,17 +230,16 @@ export const ApiController = {
       ApiController.fetchNetworkImages(),
       ApiController.fetchConnectorImages()
     ]
-    if (OptionsController.state.enableAnalytics === undefined) {
-      promises.push(ApiController.fetchAnalyticsConfig())
-    }
+
     state.prefetchPromise = Promise.race([Promise.allSettled(promises), CoreHelperUtil.wait(3000)])
   },
 
-  async fetchAnalyticsConfig() {
-    const { isAnalyticsEnabled } = await api.get<ApiGetAnalyticsConfigResponse>({
+  async fetchProjectConfig() {
+    const { isAnalyticsEnabled } = await api.get<ApiGetProjectConfigResponse>({
       path: '/getAnalyticsConfig',
```

Comment:
As I see the `/getProjectConfig` is called in tests but here we still use `/getAnalyticsConfig`. @Cali93

Reply:
It'll be renamed to `getProjectConfig` when the routes are live on prod